queenbee/recipe/dag.py

```diff
@@ -401,6 +426,13 @@ class DAGTask(BaseModel):
         description='List of inputs to loop over.'
     )
 
+    sub_folder: str = Field(
+        None,
+        description='A path relative to the current folder context where artifacts should '
+        'be saved. This is useful when performing a loop or invoking another workflow and '
+        'wanting to save results in a specific folder.'
```

Comment:
We should probably add a quick note here that this sub_folder is usually parameterized and is useful for loops similar to what you did for the key field.

Reply:
We could. It's already a pretty long description compared to the others to be honest. This is maybe something we could/should tackle in the documentation rather than API docs?